app/models/account.rb

```diff
@@ -386,6 +386,10 @@ def tenant?
     provider && !master?
   end
 
+  def api_docs_services_without_service
```

Comment:
I do not see the value of replacing `.` with a `_`

api_docs_services`.`without_service
api_docs_services`_`without_service